From 6a025e5eb379ae4b813eedba43f622abd8244ade Mon Sep 17 00:00:00 2001
From: Krisztian Litkey <kli@iki.fi>
Date: Sat, 10 Sep 2016 22:15:21 +0300
Subject: [PATCH] autogen.sh: fall back to no gtkdocize if it is there but
 fails.

---
 autogen.sh | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/autogen.sh b/autogen.sh
index 0f32089..00f5069 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -25,7 +25,13 @@ EXTRA_DIST =
 CLEANFILES =
 EOF
 else
-        gtkdocize
+        gtkdocize || {
+	rm -f gtk-doc.make
+	cat > gtk-doc.make <<EOF
+EXTRA_DIST =
+CLEANFILES =
+EOF
+}
 fi
 
 cd $olddir
-- 
2.5.5

