diff --git a/deploy.prototxt b/deploy.prototxt
index 907116e..a3de789 100644
--- a/deploy.prototxt
+++ b/deploy.prototxt
@@ -1,10 +1,11 @@
 name: "CaffeNet"
-layer {
-  name: "data"
-  type: "Input"
-  top: "data"
-  input_param { shape: { dim: 10 dim: 3 dim: 227 dim: 227 } }
-}
+
+input: "data"
+input_dim: 10
+input_dim: 3
+input_dim: 224
+input_dim: 224
+
 layer {
   name: "conv1"
   type: "Convolution"

