--- test/Makefile.in	2015-11-10 14:01:06.000000000 +0000
+++ test/Makefile.in	2015-11-30 03:12:14.150696333 +0000
@@ -1058,8 +1058,7 @@
 	unregister_filter_1.h5 unregister_filter_2.h5
 
 # Test script for error_test and err_compat
-TEST_SCRIPT = testerror.sh testlibinfo.sh testcheck_version.sh \
-	testlinks_env.sh $(am__append_1)
+TEST_SCRIPT = testerror.sh testlibinfo.sh testcheck_version.sh testlinks_env.sh $(am__append_1)
 SCRIPT_DEPEND = error_test$(EXEEXT) err_compat$(EXEEXT) \
 	links_env$(EXEEXT) $(am__append_2)
 check_SCRIPTS = $(TEST_SCRIPT)
