diff --git a/CMakeLists.txt b/CMakeLists.txt
index 08a65c4..9af14d5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -817,4 +817,10 @@ endif ()
 #-----------------------------------------------------------------------------
 configure_file (${HDF_RESOURCES_DIR}/H5pubconf.h.in ${HDF5_BINARY_DIR}/H5pubconf.h @ONLY)
 
+#-----------------------------------------------------------------------------
+# Copy the pre-generated files in place
+#-----------------------------------------------------------------------------
+configure_file(H5Tinit.c H5Tinit.c COPYONLY)
+configure_file(H5lib_settings.c H5lib_settings.c COPYONLY)
+
 include (CMakeInstallation.cmake)
